Intersect.Client.Framework/Database/GameDatabase.cs

```diff
@@ -107,6 +109,7 @@ public virtual void LoadPreferences()
             ShowExperienceAsPercentage = LoadPreference(nameof(ShowExperienceAsPercentage), true);
             ShowHealthAsPercentage = LoadPreference(nameof(ShowHealthAsPercentage), false);
             ShowManaAsPercentage = LoadPreference(nameof(ShowManaAsPercentage), false);
+            TypewriterText = LoadPreference(nameof(TypewriterText), true);
```

Comment:
Do we want it on by default? This is a legitimate question, I haven't considered it.

Reply:
I think so - I think otherwise it might be hard to tell the feature exists. It seems easier to see if it can be _disabled_ then enabled, in my opinion (ask for forgiveness instead of permission lol)

Reply:
i usually agree with having new features disabled by default, but i personally want this one to be enabled by default 🤣